README.md

```diff
@@ -259,25 +273,29 @@ Copy `$CONTAINER_EXA` to all cluster nodes (the exact path is not relevant, but
 
 You can create device files by executing (**on each node**):
 
-```console
-$ truncate -s 1G $CONTAINER_EXA/data/storage/dev.1
+```shell
+truncate -s 1G "$CONTAINER_EXA/data/storage/dev.1"
 ```
 
 This will create a sparse file of 1GB (1000 blocks of 1 MB) that holds the data. Adjust the size of the data file to your needs. Repeat this step to create multiple file devices.
 
-**IMPORTANT: Each device should be slightly bigger (~1%) than the required space for the volume(s), because a part of it will be reserved for metadata and checksums.**
+> [!IMPORTANT]
+> 
+> Each device should be slightly bigger (~1%) than the required space for the volume(s), because a part of it will be reserved for metadata and checksums.
```

Comment:
```suggestion
> Each device should be slightly larger (~1%) than the required space for the volume(s), because a part of it will be reserved for metadata and checksums.
```